mscore/inspector/inspectorBase.h

```diff
@@ -80,13 +82,28 @@ class InspectorBase : public QWidget {
 
    public:
       InspectorBase(QWidget* parent);
+      ~InspectorBase();
       virtual void setElement();
       virtual void postInit() {} // called in setElement and valueChanged
       QWidget* addWidget();
 
       friend class InspectorScriptEntry;
       };
 
+//---------------------------------------------------------
+//   InspectorScrollPreventer
+//---------------------------------------------------------
+
+class InspectorScrollPreventer : public QObject {
+      Q_OBJECT
+
+   protected:
+      bool eventFilter(QObject* watched, QEvent* event);
```

Comment:
Not sure about it, but no `override` here?

Reply:
Aha, it probably should have that (although the behaviour won't be changed). Fixed, thanks!